parsers/US_SPP.py

```diff
@@ -420,6 +430,77 @@ def fetch_exchange(
     return exchanges
 
 
+@refetch_frequency(timedelta(minutes=5))
+def fetch_realtime_price(
+    zone_key: ZoneKey,
+    session: Session | None = None,
+    target_datetime: datetime | None = None,
+    logger: Logger = getLogger(__name__),
+) -> list[dict[str, Any]]:
+    if target_datetime is None:
+        target_datetime = datetime.now(tz=timezone.utc)
+    if target_datetime.tzinfo is None:
+        target_datetime = target_datetime.replace(tzinfo=timezone.utc)
+
+    closest_5_minutes_datetime = get_closest_5_minutes_datetime(target_datetime)
+    url = f"{REALTIME_PRICE_URL}&path=/{closest_5_minutes_datetime.strftime('%Y')}/{closest_5_minutes_datetime.strftime('%m')}/By_Interval/{closest_5_minutes_datetime.strftime('%d')}/RTBM-LMP-SL-{closest_5_minutes_datetime.strftime('%Y%m%d%H%M')}.csv"
```

Comment:
This is a gnarly format string -- maybe extract to a helper function or join them into a path beforehand for readability.